internal/config/config.go

```diff
@@ -52,6 +52,15 @@ type Config struct {
 	//
 	// ArtifactRoot is specified with the -artifact-root flag.
 	ArtifactRoot string
+	// Source is the path to the root of the googleapis repository.
+	// When this is not specified, the googleapis repository is cloned
+	// automatically.
+	//
+	// Source is used by generate, update-apis, update-image-tag and configure
+	// commands.
+	//
+	// Source is specified with the -source flag.
+	Source string
```

Comment:
This might be from a bad merge? This field already exists below.

Reply:
@codyoss 
Apologies for the bad merge. I have fixed it.